src/protocols/types/match_making.js

```diff
@@ -508,16 +501,22 @@ class JoinMatchmakeSessionParam extends NEXTypes.Structure {
 		this.strSystemPassword = stream.readNEXString();
 		this.joinMessage = stream.readNEXString();
 		this.participationCount = stream.readUInt16LE();
-		this.extraParticipants = stream.readUInt16LE();
 
-		// TODO - Make this better. Dirty hack to skip this check on Minecraft: Wii U Edition
-		if (stream.connection.title.access_key !== 'f1b61c8e') {
+		// * Assuming this to be 3.10.0
+		// * Not seen in Terraria, which is 3.8.3
+		if (semver.gte(nexVersion, '3.10.0')) {
```

Comment:
Just for future reference, Minecraft has structure version 1 on `JoinMatchmakeSessionParam`

Reply:
Is this a structure difference then and not a version difference?

Reply:
I'm not sure, maybe the structure revision came with a specific NEX version. This could be like on `MatchmakeSession` fields, where on some NEX versions the structure revision is increased. I've only said this as additional information